### PR TITLE
fix: deploy mintable digital assets

### PR DESCRIPTION
**Pull request by CallumGrindle**
### What kind of change does this PR introduce (bug fix, feature, docs update, ...)?
fix

### What is the current behaviour (you can also link to an open issue here)?
currently deployed digital asset contracts are not implementations. 

### What is the new behaviour (if this is a feature change)?
Now deploys mintable digital asset contracts with external mint function

### CODE REVIEW

**Review by rryter — COMMENTED** [review]
Looks good. 

Are you still blocked with that problem in the tests?

**Comment by CallumGrindle** [comment]
> Looks good.
> 
> Are you still blocked with that problem in the tests?

Yep still blocked by the tests issue atm. Want to spend time to improve the tests this week

**Review by Hugoo — APPROVED** [review]
cool

will be a breaking change because the name of LSP7 and LSP8 has been updated right?

File: src/lib/classes/digital-asset.ts [hunk 1]
```diff
@@ -72,9 +72,7 @@ export class DigitalAsset {
       this.signer,
       digitalAssetDeploymentOptions,
       contractDeploymentOptions?.libAddress ??
-        versions[this.options.chainId]?.baseContracts?.LSP8IdentifiableDigitalAsset[
-          DEFAULT_CONTRACT_VERSION
-        ]
+        versions[this.options.chainId]?.baseContracts?.LSP8Mintable[DEFAULT_CONTRACT_VERSION]
```
[the remark this answers is not in the document]
Comment (CallumGrindle) [reply src/lib/classes/digital-asset.ts:75]: The versions file is internal to the lib so is always there. If it doesn't match and cant find a base contract address (e.g. if a user passes a different chainId for a local network) then it defaults to deploying a normal contract without using proxy/base contract pattern.

Also no breaking change as all name changes are internal :)

**Pull request merged**